components/esp_modem/include/cxx_include/esp_modem_exception.hpp

```diff
@@ -16,51 +16,73 @@
 
 #include <string>
 #include "esp_err.h"
+#include "esp_log.h"
+
+#ifndef __FILENAME__
+#define __FILENAME__    __FILE__
+#endif
+#define ESP_MODEM_THROW_IF_FALSE(...) esp_modem::throw_if_false(__FILENAME__, __LINE__, __VA_ARGS__)
+#define ESP_MODEM_THROW_IF_ESP_FAIL(...) esp_modem::throw_if_error(__FILENAME__, __LINE__, __VA_ARGS__)
```

Comment:
```suggestion
#define ESP_MODEM_THROW_IF_ERRORL(...) esp_modem::throw_if_error(__FILENAME__, __LINE__, __VA_ARGS__)
```

Reply:
Sorry, missed the macro name.

Reply:
I think this doesn't fit very well to the macro names, as they actually `trow` if the return value is `ESP_FAIL` exactly.

Maybe `ESP_MODEM_THROW_IF_RETURS_ESP_FAIL()`?

Reply:
In `esp_modem::throw_if_error` implementation, we throw if != `ESP_OK` that's why I suggested the name change.

Reply:
oh, you're right. Updated!